components/button/button.css

```diff
@@ -0,0 +1,110 @@
+.btn {
+  position: relative;
+  display: inline-block;
+  text-align: center;
+  min-width: 150px;
+  padding: 0.8rem 1.5rem;
+  font-size: 1rem;
+  margin: 1rem;
+  border: none;
+  cursor: pointer;
+  border-radius: 2px;
+  font-weight: var(--font-semibold);
+  color: white;
+  text-transform: uppercase;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.1);
```

Comment:
Just a suggestion you can use variables here as I was also having a bit problem writing the whole shadow color like this
only suggesting apply if you are comfortable
![Screenshot (442)](https://user-images.githubusercontent.com/75560307/153179425-ee073faf-2fb4-4b40-a62a-9d1959397f3e.png)

Reply:
Yeah, I understand, the only thing is I'm not using the same box-shadows everywhere and thus using the hardcoded values so that I don't create variables that will be used only once.